.github/workflows/docker_linux.yml

```diff
@@ -39,25 +39,33 @@ jobs:
   push:
     runs-on: ubuntu-latest
     env:
-      DOCKER_BUILDKIT: 1
```

Comment:
If you are going to use these actions consider also using this which let's us get parallel builds for the different build containers. 
https://github.com/docker/setup-buildx-action

Reply:
I had it on my workflow but removed from this one to keep it simple.  I'll add it again, thanks.